tox.ini

```diff
@@ -1,8 +1,16 @@
 [tox]
-envlist = pypy, pypy3, py27, py34, py35, py36
+envlist = pypy, pypy3, py27, py34, py35, py36, flake8
 
 [testenv]
 deps = -rrequirements-dev.txt
 commands =
     py.test sasstests.py
+
+[testenv:flake8]
+deps= flake8
+skip_install=True
+commands=
     flake8 .
```

Comment:
Some numbers so we can quantify this:

## `.tox/py27/bin/flake8 .`
- `real	0m0.892s`
- `real	0m0.896s`
- `real	0m0.889s`

Let's call this 1 second

## `tox -e py27` (without flake8)
- `real	2m0.758s`
- `real	2m4.935s`
- `real	1m59.932s`

Let's say ~120s

## `tox -e flake8`
- `real	0m5.010s`
- `real	0m5.033s`
- `real	0m5.034s`

So let's say that's ~5s

## Extrapolating a bit

(I removed the pypy3 environment in my testing, it's broken due to setuptools / pip)

Given we have 5 environments (pypy is significantly slower, but we'll assume it's the same speed as py27, the py3 environments are also slightly slower for whatever reason but for simplicity we'll assume they're the same).

**Current time:** 120s * 5 + 1s * 5 => 605s
**New time:** 120s * 5 + 5s => 605s

So basically no savings -- if anything it's now a fraction of a second slower (since I rounded the `tox -e flake8` time down and rounded the `.tox/py27/bin/flake8 .` time up).  This time savings is entirely irrelevant compared to the 10 minutes of building C bits though :)